jackson/src/main/java/feign/jackson/JacksonDecoder.java

```diff
@@ -66,4 +69,23 @@ public Object decode(Response response, Type type) throws IOException {
       throw e;
     }
   }
+
+  private Charset getResponseCharset(Response response) {
```

Comment:
I recommend moving this to `Response#getCharset()`.  This way it can be used by all decoders.

Reply:
I agree, but maybe the method name should be Response#charset(), to follow naming convention in Response (Response#body(), Response#headers(), etc)